### PR TITLE
Deprecate move cell up/down

### LINKED ISSUE #793: Move cell up and down inconsistant.
do not work with multiple selection (we can decide that how it works though), 
but imho it should move blocks. 

Or implementation side, move the cell before the block after the block when move up,  and vice versa. (Though it might unbind callbacks)

### PR DESCRIPTION
**Pull request by jdfreder**
~~closes #793~~
workaround for #793

### CODE REVIEW

**Comment by Carreau** [comment]
> closes #793

Well no it does not closes, move up-down will still be broken, but I'm fine deprecating. 
Though it will still stays all through the 4.x series, and all will break for 5.0+.

Can you though expand the deprecation description to say since when it's deprecated, (4.1) and when it will be removed (5.0) ?

Thanks !

**Comment by jdfreder** [comment]
Sure, see last commit

File: notebook/static/notebook/js/notificationarea.js [hunk 1]
```diff
@@ -305,6 +305,9 @@ define([
         this.events.on('notebook_copy_failed.Notebook', function (evt, error) {
             nnw.warning(error.message || "Notebook copy failed");
         });
+        this.events.on('deprecated.Notebook', function (evt, message) {
+            nnw.warning(message, 2000);
+        });
```
Comment (Carreau) [inline notebook/static/notebook/js/notificationarea.js]: I would do that, deprecation should be show only to devs.

Comment (jdfreder) [inline notebook/static/notebook/js/notificationarea.js]: Would not* do that?

Comment (Carreau) [inline notebook/static/notebook/js/notificationarea.js]: Yep. Fixed.

Comment (jdfreder) [inline notebook/static/notebook/js/notificationarea.js]: Removed in latest commit

**Comment by Carreau** [comment]
Merging once travis go green. 
We will get 4.1 beta by Friday !

**Comment by jasongrout** [comment]
Is there a replacement for this? Apparently it's being used (@williamstein uses it quite a lot in SageMathCloud, for example).

**Comment by williamstein** [comment]
I now retract my need for this -- I thought we were using it in SMC, but we're not.

**Pull request merged**